python-sdk/nuscenes/eval/detection/render.py

```diff
@@ -28,6 +28,7 @@ def visualize_sample(nusc: NuScenes,
                      conf_th: float = 0.15,
                      eval_range: float = 50,
                      verbose: bool = True,
+                     display_legend: bool = True,
```

Comment:
`display_legend` should be `False` by default to be backwards-compatible

Reply:
have done